cmd/app.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const GetScript = "curl -sfL https://get.k3s.io"
```

Comment:
If this is only used in the `cmd` package then it should have a lowercase (sorry if I was confusing) `getScript` - otherwise it becomes a global available outside the package.

Reply:
My bad, will fix this.